plugins/rikki/heroeslounge/components/UpdateMatch.php

```diff
@@ -97,10 +97,11 @@ public function onRescheduleMatch()
                         $tbp = new DateTime($match->tbp, new DateTimeZone(TimezoneHelper::defaultTimezone()));
                         $tbp->setTimezone(new DateTimeZone($timezone));
                         Flash::error('The match has to be played before ' . $tbp->format('d M Y H:i'));
-                    } else {
-                        $wbp->setTimezone(new DateTimeZone($timezone));
-                        Flash::error('The new schedule time ' . $wbp->format('d M Y H:i') . ' can not be in the past');
+                        return;
```

Comment:
This is still the same nesting as before, and it also returns the wrong thing

Reply:
I think I misunderstood originally, will fix.